webots_ros2_driver/src/WebotsNode.cpp

```diff
@@ -39,6 +39,8 @@ namespace webots_ros2_driver
   const char *gPluginInterface = "webots_ros2_driver::PluginInterface";
   const char *gPluginInterfaceName = "webots_ros2_driver";
 
+  bool WebotsNode::mShutdownSignalReceived = false;
```

Comment:
This shouldn't be exposed:
```suggestion
  bool gShutdownSignalReceived = false;
  
  void handleSigint()
  {
    bool gShutdownSignalReceived = true;
  }
```